recipe/meta.yaml

```
@@ -63,24 +63,24 @@ outputs:
         - cudatoolkit      # [(cuda_compiler_version or "").startswith("11")]
         - cuda-cudart-dev  # [(cuda_compiler_version or "").startswith("12")]
         - nccl             # [cuda_compiler_version != "None"]
-        - librmm ={{ rapids_version }}  # [linux]
+        - librmm >={{ rapids_version }}  # [linux and cuda_compiler_version != "None"]
```

So `=`s already implies the patch version can float

Do we want to keep the same major minor version here?

```suggestion
        - librmm ={{ rapids_version }}  # [linux and cuda_compiler_version != "None"]
```

---

Fixed in the latest commit

---

Thanks! 🙏

Marking these 2 resolved